zimbra/os-requirements/RHEL.def

```diff
@@ -1,2 +1,3 @@
-osdeps.RHEL7_64 := nmap-ncat
+osdeps.RHEL8_64 := nmap-ncat, spax
+osdeps.RHEL7_64 := nmap-ncat, pax
```

Comment:
I think these are build time dependencies
not sure we need pax here

Reply:
although these same deps are part of specfile, but it's confusing
can we add it in https://github.com/Zimbra/packages/blob/3fdcd1ecb8a9758e6516227ffd2482909e9a940a/zimbra/os-requirements/zimbra-os-requirements/rpm/SPECS/os-requirements.spec#L8 directly?